Gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+
+gem "fastlane", "~> 2.224.0"
+gem 'abbrev'
+gem 'logger'
+gem 'mutex_m'
+gem 'csv'
```

Comment:
Do you know why these are now required to have here?

Reply:
Yeah. I could not run `bundle exec fastlane` without these.

https://github.com/fastlane/fastlane/issues/21942#issuecomment-2585471396